doc/nrf/images/bt_mesh_device_lifecycle.svg

```diff
@@ -2,93 +2,89 @@
+		.st2 {fill:#ffffff;font-family:Calibri;font-size:1.00001em}
 		.st3 {font-size:1em}
-		.st4 {fill:#dbeef3;stroke:#000000;stroke-linecap:round;stroke-linejoin:round;stroke-opacity:0;stroke-width:0.24}
-		.st5 {fill:#f2f2f2;stroke:#000000;stroke-linecap:round;stroke-linejoin:round;stroke-opacity:0;stroke-width:0.24}
-		.st6 {marker-end:url(#mrkr5-24);stroke:#000000;stroke-dasharray:5.25,3.75;stroke-linecap:round;stroke-linejoin:round;stroke-width:0.75}
-		.st7 {fill:#000000;fill-opacity:1;stroke:#000000;stroke-opacity:1;stroke-width:0.22935779816514}
-		.st8 {marker-start:url(#mrkr5-35);stroke:#000000;stroke-dasharray:5.25,3.75;stroke-linecap:round;stroke-linejoin:round;stroke-width:0.75}
-		.st9 {stroke:#000000;stroke-dasharray:0,3.75;stroke-linecap:round;stroke-linejoin:round;stroke-width:0.75}
-		.st10 {fill:none;stroke:none;stroke-linecap:round;stroke-linejoin:round;stroke-width:0.75}
+		.st4 {fill:#0033a0;stroke:none;stroke-width:1}
+		.st5 {marker-end:url(#mrkr13-18);stroke:#333f48;stroke-dasharray:8,4;stroke-linecap:butt;stroke-width:1}
+		.st6 {fill:#333f48;fill-opacity:1;stroke:#333f48;stroke-opacity:1;stroke-width:0.28409090909091}
+		.st7 {fill:#d9e1e2;stroke:none;stroke-width:1}
+		.st8 {fill:#333f48;font-family:Calibri;font-size:1.00001em}
+		.st9 {stroke:#333f48;stroke-dasharray:1,4;stroke-linecap:butt;stroke-width:1}
+		.st10 {fill:none;stroke:none;stroke-width:1}
 		.st11 {fill:none;fill-rule:evenodd;font-size:12px;overflow:visible;stroke-linecap:square;stroke-miterlimit:3}
 	]]>
 	</style>
 
 	<defs id="Markers">
-		<g id="lend5">
-			<path d="M 2 1 L 0 0 L 1.98117 -0.993387 C 1.67173 -0.364515 1.67301 0.372641 1.98465 1.00043 " style="stroke:none"/>
+		<g id="lend13">
+			<path d="M 3 1 L 0 0 L 3 -1 L 3 1 " style="stroke:none"/>
 		</g>
-		<marker id="mrkr5-24" class="st7" refX="-7.63" orient="auto" markerUnits="strokeWidth" overflow="visible">
-			<use xlink:href="#lend5" transform="scale(-4.36,-4.36) "/>
-		</marker>
-		<marker id="mrkr5-35" class="st7" refX="7.15" orient="auto" markerUnits="strokeWidth" overflow="visible">
-			<use xlink:href="#lend5" transform="scale(4.36) "/>
+		<marker id="mrkr13-18" class="st6" refX="-10.56" orient="auto" markerUnits="strokeWidth" overflow="visible">
+			<use xlink:href="#lend13" transform="scale(-3.52,-3.52) "/>
 		</marker>
 	</defs>
 	<g>
 		<title>Page-1</title>
-		<g id="shape19-1" transform="translate(0.12,-78.7429)">
-			<title>Process</title>
+		<g id="shape1032-1" transform="translate(140.093,-146.592)">
+			<title>Nordic Blue.1032</title>
+			<desc>Provisioned mesh network mode</desc>
+			<rect x="0" y="262.979" width="102.829" height="49.9973" class="st1"/>
+			<text x="8.16" y="284.38" class="st2">Provisioned mesh <tspan x="15.54" dy="1.2em" class="st3">network mode</tspan></text>		</g>
+		<g id="shape1033-5" transform="translate(279.186,-81.0407)">
+			<title>Nordic Blue.1033</title>
+			<desc>Configure mesh network mode</desc>
```

Comment:
The original image said "Configured mesh network node"

Reply:
Good catch, this is now fixed.